lib/mutant/mutator/node/zsuper.rb

```diff
@@ -16,6 +16,7 @@ class ZSuper < self
         # @api private
         def dispatch
           emit_singletons
+          emit(N_EMPTY_SUPER)
```

Comment:
I wounder if we should rename this to `N_ZSUPER` some day to match the AST types better.